go/vt/vttablet/tabletmanager/vreplication/vcopier.go

```diff
@@ -388,6 +388,9 @@ func (vc *vcopier) copyTable(ctx context.Context, tableName string, copyState ma
 	defer vc.vr.stats.PhaseTimings.Record("copy", time.Now())
 	defer vc.vr.stats.CopyLoopCount.Add(1)
 
+	if err := vc.vr.setState(binlogdatapb.VReplicationWorkflowState_Copying, ""); err != nil {
+		return err
+	}
```

Comment:
See also:

https://github.com/vitessio/vitess/blob/11edc633c0f8f4163e04c9de3f548fc662c7a5b0/go/vt/vttablet/tabletmanager/vreplication/vcopier.go#L246-L248

in `initTablesForCopy`, which explains why the first run's status shows as Copying, while resuming shows Running (even though we're still copying).